# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1301773276 25200
#      Sat Apr 02 12:41:16 2011 -0700
# Node ID b22816427ce92ae94439e0e42e7ca4c1cee26e48
# Parent  cc8ccdfec424b013d0f885919afb53d8394cb6d4
Add isdeployed function to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2011-04-02  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/system.txi: Add isdeployed function to documentation.
+
 2011-03-31  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/io.txi: Add list_in_columns, terminal_size functions
 	to documentation.
 
 2011-03-31  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/errors.txi, interpreter/octave.texi: Add onCleanup
diff --git a/doc/interpreter/system.txi b/doc/interpreter/system.txi
--- a/doc/interpreter/system.txi
+++ b/doc/interpreter/system.txi
@@ -477,16 +477,18 @@ referred to as a @var{grp_struct}.
 @DOCSTRING(ispc)
 
 @DOCSTRING(isunix)
 
 @DOCSTRING(ismac)
 
 @DOCSTRING(isieee)
 
+@DOCSTRING(isdeployed)
+
 @DOCSTRING(OCTAVE_HOME)
 
 @DOCSTRING(matlabroot)
 
 @DOCSTRING(OCTAVE_VERSION)
 
 @DOCSTRING(version)
 
